app/components/EventItem/index.tsx

```diff
@@ -78,16 +78,12 @@ const TimeStamp = ({ event }: TimeStampProps) => {
         <Icon
           name="calendar-number-outline"
           size={20}
-          style={{ cursor: 'pointer', marginRight: '10px' }}
```

Comment:
These were removed because it doesn't make sense to be able to click on these icons.

Reply:
We should probably add a `pointer` prop or something, since I see inline styling like this everywhere. But that can be done later!

Reply:
Yeah, I agree! Thought about it, but I'll rather create a separate issue for it :ok_hand: